training/distributed_training/pytorch/model_parallel/gpt-j/11_train_gptj_smp_tensor_parallel_notebook.ipynb

```diff
@@ -39,7 +39,7 @@
     "! pip install -qU \"sagemaker>=2,<3\"\n",
```

Comment:
<p>Change the title to:</p><p> </p><h3>Create a SageMaker PyTorch Estimator</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>